language/snippets/src/test/java/com/example/language/AnalyzeIT.java

```diff
@@ -19,8 +19,8 @@
 import static com.google.common.truth.Truth.assertThat;
 
 import com.google.cloud.language.v1.PartOfSpeech.Tag;
```

Comment:
nit: what is a reason to keep v1 import?

Reply:
This is for `AnalyzeSyntax` API which is still v1 API. We need to keep it until V2 is available.